.github/workflows/ci.yml

```diff
@@ -42,7 +42,7 @@ jobs:
           - 'swift:5.7'
           - 'swift:5.8'
           - 'swift:5.9'
-          - 'swiftlang/swift:nightly-5.10-jammy'
+          #- 'swiftlang/swift:nightly-5.10-jammy'
```

Comment:
Nightly is crashing with code coverage. I need to add a bug in apple/swift for that.